test/jdk/java/net/httpclient/http2/ExpectContinueResetTest.java

```diff
@@ -88,15 +89,19 @@ public Object[][] testData() {
 
     @Test(dataProvider = "testData")
     public void test(URI uri) {
+        out.printf("\nTesting with Version: %s, URI: %s\n", HTTP_2, uri.toASCIIString());
         err.printf("\nTesting with Version: %s, URI: %s\n", HTTP_2, uri.toASCIIString());
         Iterable<byte[]> iterable = EndlessDataChunks::new;
         HttpRequest.BodyPublisher testPub = HttpRequest.BodyPublishers.ofByteArrays(iterable);
+        Exception exception = null;
         Throwable testThrowable = null;
         try {
             performRequest(testPub, uri);
         } catch (Exception e) {
+            exception = e;
             testThrowable = e.getCause();
         }
+        assertNotNull(exception, "Request should have completed exceptionally but exception is null");
```

Comment:
Unlike the `assertNotNull` on the next line, I think it might be better to replace this new `assertNotNull` with a `fail(...)` inside the try block immediately after the `performRequest()` call.